static/src/js/containers/DatepickerContainer/DatepickerContainer.jsx

```diff
@@ -139,7 +139,7 @@ class DatepickerContainer extends Component {
     // A valid date will come be passed as an ISO string. Check to see if the date is a valid ISO string,
     // if so, we convert it to a UTC string in our desired format. If the value is not a valid ISO date,
     // then we leave it untouched and pass it to the input.
-    const isValidISO = moment.utc(value, moment.ISO_8601, true).isValid()
+    const isValidISO = moment.utc(value, 'YYYY-MM-DDTHH:m:s.SSSZ', true).isValid()
```

Comment:
Please add a comment explaining why we are using this string over `moment.ISO_8601`, which seems like the better option, so that we don't change it back in the future unexpectedly.

Reply:
We might end up changing it in the future...

Reply:
> so that we don't change it back in the future unexpectedly.

If we change code in the future we update our documentation or comments when we do. This comment would hopefully fix a future issue like we saw with this bug.